our-blog.md

```diff
@@ -0,0 +1,50 @@
+# Our Blog
+
+We currently maintain a Medium Publication called [AnitaB.org Open Source](https://medium.com/anitab-org-open-source).
+
+**Table of Content:**
+
+- [Main Goals of Blogging](#main-goals-of-blogging)
+- [How to submit to our Publication](#how-to-submit-to-our-publication)
+- [Blog History](#blog-history)
+
+
+## Main Goals of Blogging
+
+- Promote Open Source
+    - This includes people outside of our open source community.
+- Promote Community Activities/Projects
+    - This includes our members, projects, programs, and events.
+- Debunk Myths of Contribution == Code
+    - This includes contributions of documentation, design, testing, and outreach.
+- Elevate AnitaB Brand
+    - This includes everything occurring in the community outside of GHC events.
+
+## How to submit to our Publication
+
+To write in our publication you nneed to be admitted as a writer on AnitaB.org Open Source Medium Publication. If you are not already, please let us know on [#outreach-research](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research) Zulip stream.
```

Comment:
Can we be a bit more specific for eg: 
If you are not already, please let us know in [#outreach-research](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research) Zulip stream. Make sure to provide us with following details : 
* The title and sub-title of the blog (even if the blog is in progress)
* How does it convey our main goals of blogging 
* Your medium handle (@ )

(Although this will help us it might intimidate the blogger if a newcomer)

Reply:
That is a very interesting comment @siddhipandare ! I feel the medium handle is a must, the title and sub-title its also good, because it requires them to be added already with an idea in mind, instead of adding as a writer just because.

Regarding "How does it convey our main goals of blogging" that seems appropriate and fair to ask, but it does seem to be intimidating for a newcomer. I will add it, because from reading the main goal, it may not be to hard to do it.

Reply:
I asked just the title instead of title and subtitle

Reply:
Okay the title is good enough for us 😄